core/sugarConstructors.ml

```diff
@@ -2,18 +2,15 @@ open Sugartypes
 open Operators
 open Utility.OptionUtils
 
-module type Pos = sig
-  type t
-  val pos      : t -> Sugartypes.position
-  val with_pos : t -> 'a -> 'a Sugartypes.with_pos
-end
-
-module type SmartConstructorsSig = SmartConstructorsIntf.SmartConstructorsSig
+(* Import module signatures *)
```

Comment:
Missing punctuation.

Reply:
I tend not to short (single-sentence) comments with punctuation - that's why it's consistently missing. Will fix.